doc/sphinx/language/core/sections.rst

```diff
@@ -66,12 +66,20 @@ usable outside the section as shown in this :ref:`example <section_local_declara
    In this case, the proof should be terminated with :cmd:`Defined` in order to define a constant
    for which the computational behavior is relevant.  See :ref:`proof-editing-mode`.
 
+.. attr:: clearbody
+
+   When used with :cmd:`Let` in a section,
+   clears the body of the definition in the proof context of following proofs.
+
 .. warn:: @ident is declared opaque but this is not fully respected inside the section and not at all outside the section.
 
-   Terminating the proof for a :cmd:`Let` with :cmd:`Qed` with `Let` is not recommended.
+   Terminating the proof for a :cmd:`Let` with :cmd:`Qed` is deprecated.
```

Comment:
```suggestion
   Terminating the proof for a :cmd:`Let` with :cmd:`Qed` is deprecated; the proof
   should be terminated with :cmd:`Defined`.  Use :attr:`clearbody` with :cmd:`Defined`
   to clear the body of the proof when the section is closed.
```

and adjust the following lines (77-82).

Also, would you move the part about "The variable will appear to have no body inside proofs but the kernel will still use the body when checking" into the description of `clearbody`" if it is specific to `clearbody`?  If it's standard behavior for `Qed` (just opaqueness?) we shouldn't mention that important fact only in the description of a warning.  If it's just opaqueness, maybe use that term in the description.